posts/templates/posts/post_detail.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-xs-0 col-sm-3 col-md-4"></div>
+        <div class="col-xs-12 col-sm-6 col-md-4">
```

Comment:
This seems to be to small in width. It would be nice to make it as wide as everywhere else.
![screenshot from 2015-06-05 21-01-12](https://cloud.githubusercontent.com/assets/872251/8013043/3ff06d04-0bc6-11e5-9953-ddcd5ae13e1b.png)